tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -1,1165 +1,44 @@
-                "text"
-            ]
-        },
-        {
-            "Kind": 0,
-            "Value": ",",
-            "RenderClasses": [
-                "punc"
-            ]
-        },
-        {
-            "Kind": 4,
-            "Value": ""
-        },
-        {
-            "Kind": 0,
-            "Value": "string",
-            "RenderClasses": [
-                "keyword"
-            ]
-        },
-        {
-            "Kind": 2,
-            "Value": ""
-        },
-        {
-            "Kind": 0,
-            "Value": "blobContainerName",
-            "RenderClasses": [
-                "text"
-            ]
-        },
-        {
-            "Kind": 0,
-            "Value": ",",
-            "RenderClasses": [
-                "punc"
-            ]
-        },
-        {
-            "Kind": 4,
-            "Value": ""
-        },
-        {
-            "Kind": 0,
-            "Value": "string",
-            "RenderClasses": [
-                "keyword"
-            ]
-        },
-        {
-            "Kind": 2,
-            "Value": ""
-        },
-        {
-            "Kind": 0,
-            "Value": "blobName",
-            "RenderClasses": [
-                "text"
-            ]
-        },
-        {
-            "Kind": 0,
-            "Value": ")",
-            "RenderClasses": [
-                "punc"
-            ]
-        },
-        {
-            "Kind": 0,
-            "Value": ";",
-            "RenderClasses": [
-                "punc"
-            ]
-        },
-        {
-            "Kind": 1,
-            "Value": ""
-        }
-    ]                          
-  ```
-</details>
-
-### Navigation Token
+A sample token file for .NET package `Azure.Template` is present [here](./apiview-treestyle-parser-schema/sample/Azure.Template_token.json) and corresponding rendered text representation is available [here](./apiview-treestyle-parser-schema/sample/Azure.Template_review_content.txt).
 
-Make a token a navigation token by adding key `NavigateToId` to the token property with a value corresponding the the Id of the APINode to navigate to on click.
-<details>
-<summary>click to expand json</summary>
 
-  ```json
-    {
-        "Kind": 0,
-        "Value": "BlobClientOptions",
-        "Properties": {
-            "NavigateToId": "Azure.Storage.Blobs.BlobClientOptions"
-        },
-        "RenderClasses": [
-            "tname"
-        ]
-    }                         
-  ```
-</details>
+## JSON token validation
 
-### URL Token
-![Navigation](images/url-token.png)
+You can validate JSON tokens against required JSON schema using [JSON schema validator](https://www.jsonschemavalidator.net/).
 
-Make a token a Url by adding key `NavigateToUrl` to the token property and setting the value to the target Url.
-<details>
-<summary>click to expand json</summary>
+- Select `Custom` as schema type and copy and paste the contents in [json schema](./apiview-treestyle-parser-schema/model.json) to left panel.
```

Comment:
Jonathan was saying we can publish the json schema on https://www.schemastore.org/json/
or somewhere and that way people can just reference and there is no need for them to manually copy it every time they want to validate it.
Look in Teams for the `APIView Documentation` group chat. Jonathan sent the message on 7/23

Reply:
I have discussed this with Jonathan and he is fine with JSON schema stored in azure-sdk-tools repo.